lib/utils/job-utils/net-snmp-tool.js

```diff
@@ -35,7 +35,7 @@ function SnmpFactory(assert, parser, ChildProcess, Promise, _) {
         }
         var outputFormat = options.numericOutput ? '-Oqn' : '-Oq';
         // -Oq makes parsing easier by simplifying snmptool output
-        var args = [outputFormat, '-v2c', '-c', this.community, this.host].concat(oid.split(' '));
+        var args = [outputFormat, '-v2c', '-t120', '-c', this.community, this.host].concat(oid.split(' '));
```

Comment:
I think it's better to add the timeout in the `options` and let it default to 120 rather than hard-code. So it's flexible if other command want another timeout value.
```
if (!options.timeout && options.timeout !== 0) {
    options.timeout = 120;
}
var args = [outputFormat, '-v2c', '-t' + options.timeout.toString(), '-c', this.community, this.host].concat(oid.split(' '));
```